cmd/containerd/main.go

```diff
@@ -110,6 +112,10 @@ func main() {
 			return err
 		}
 
+		//publish profiling and debug info.
+		log.G(ctx).WithField("socket", path).Info("starting profiler handlers")
+		go serveProfiler(ctx, l)
```

Comment:
This listener is supposed to be for the grpc server.

There should be a separate unix socket for each of metrics, debug and grpc.